VideoAPI/VideoApi.Domain/Conference.cs

```diff
@@ -175,5 +177,15 @@ public Participant GetVideoHearingOfficer()
         {
             return Participants.SingleOrDefault(x => x.IsVideoHearingOfficer());
         }
+
+        public IList<Message> GetMessages()
+        {
+            return Messages.OrderByDescending(x => x.TimeStamp).ToList();
+        }
+        public void AddMessage(string from, string to, string messageText)
+        {
+            var message = new Message(from, to, messageText);
+            Messages.Add(message);
```

Comment:
There should be a clear messages function too (including tests)

Reply:
Done